packages/dev/core/src/Rendering/IBLShadows/iblShadowsVoxelRenderer.ts

```diff
@@ -527,10 +533,26 @@ export class _IblShadowsVoxelRenderer {
                 }
             },
         });
-        voxelMaterial.cullBackFaces = false;
-        voxelMaterial.backFaceCulling = false;
-        voxelMaterial.depthFunction = Engine.ALWAYS;
-        return voxelMaterial;
+        this._voxelMaterial.cullBackFaces = false;
+        this._voxelMaterial.backFaceCulling = false;
+        this._voxelMaterial.depthFunction = Engine.ALWAYS;
+
+        this._voxelSlabDebugMaterial = new ShaderMaterial("voxelSlabDebug", this._scene, "iblVoxelSlabDebug", {
+            uniforms: ["world", "viewMatrix", "cameraViewMatrix", "projection", "invWorldScale", "nearPlane", "farPlane", "stepSize"],
+            defines: ["MAX_DRAW_BUFFERS " + this._maxDrawBuffers],
+            shaderLanguage: isWebGPU ? ShaderLanguage.WGSL : ShaderLanguage.GLSL,
+            extraInitializationsAsync: async () => {
+                if (isWebGPU) {
+                    await Promise.all([import("../../ShadersWGSL/iblVoxelSlabDebug.fragment"), import("../../ShadersWGSL/iblVoxelSlabDebug.vertex")]);
+                } else {
+                    await Promise.all([import("../../Shaders/iblVoxelSlabDebug.fragment"), import("../../Shaders/iblVoxelSlabDebug.vertex")]);
+                }
+            },
+        });
+        this._scene.onBeforeRenderObservable.add(() => {
+            this._voxelSlabDebugMaterial.setMatrix("projection", this._scene.activeCamera!.getProjectionMatrix());
+            this._voxelSlabDebugMaterial.setMatrix("cameraViewMatrix", this._scene.activeCamera!.getViewMatrix());
+        });

```

Comment:
You should do this only when debugging is turned on (and remove the observers - in case they have been added - when debugging is turned off).

Reply:
Done